### ISSUE TITLE
line anchor in multi-line label based on first line

### ISSUE THREAD

**Issue by anne-gropler** [cginternals/webgl-operate#171]
The line anchor (Label.LineAnchor) only applies to the first line of a label. Given a multi-line label, the line anchor `Centered` won't vertically center the label at the given point.

Is this desired behaviour, since it is called `Lineanchor` and not `LabelAnchor`?

Reproduce this at the Label 3D example on branch `example-3d-label`.

**Comment by anne-gropler** [comment]
![grafik](https://user-images.githubusercontent.com/5690856/54205975-afd2dc80-44d7-11e9-8d91-1130c366ce74.png)
Expected the plus ('+') glyph to be more in the vertical middle of the 'target cross'.

**Comment by cgcostume** [comment]
Yes this is intended. But we probably should add this, as you suggested, by introducing `LabelAnchor`.
